mne/stats/regression.py

```diff
@@ -246,13 +246,12 @@ def linear_regression_raw(raw, events, event_id=None, tmin=-.1, tmax=1,
     """
 
     if isinstance(solver, string_types):
-        if solver == 'pinv':
-            fast_dot = _get_fast_dot()
+        if solver == 'cholesky':
+            def solver(X, y):
+                a = (X.T * X).toarray()
```

Comment:
I prefer `X.T.dot(X)`, since it makes clear it's a dot product. Otherwise I have to try and assume/figure it outfrom the code. 99.9% (or 100%) of other times in our code `*` means broadcast-multiply, not dot.